crypto/src/proteus.rs

```diff
@@ -93,6 +93,15 @@ impl CoreCrypto {
         Ok(())
     }
 
+    /// Reloads the sessions from the key store
+    ///
+    /// Warning: The Proteus client **MUST** be initialized with [CoreCrypto::proteus_init] first or an error will be returned
+    pub async fn proteus_reload_sessions(&mut self) -> CryptoResult<()> {
```

Comment:
Does this mean that it will throw an error if I run an mls only instance of CC and call `restoreFromDisk`?  What about just making it a no-op if proteus is not initialised?

Reply:
Yes, we do on Android.

Reply:
And we only have one type of binary which has MLS and proteus enabled.

Reply:
Ok. If it isn't initialised it will do nothing instead of failing